src/test/scala/com/peoplepattern/text/StringUtilSpec.scala

```diff
@@ -0,0 +1,184 @@
+package com.peoplepattern.text
+
+import org.scalatest._
+import java.net.URL
+
+class StringUtilSpec extends FlatSpec {
```

Comment:
We don't have that method defined for `Option[String]`, though it's not a bad idea.